### PR TITLE
created the forgot password form component and page and integrated it…

### PR DESCRIPTION
**Pull request by AdekolaToniloba**
… with the backend endpoints. Integrated the signup form with the backend endpoints as well
# **Description**
This pr implements the forgot password feature page among other things. The pr also integrates the signup form with the API endpoints.
### Figma Link
[FIGMA](https://www.figma.com/design/7DGLgn48eyVicv8JpkPchw/dRello?node-id=529-3918&node-type=frame&t=niMC88w4oNmkZjzP-0)

---

# **What were you told to do?**
Acceptance Criteria:
- User should be able to input email address.
- User should be able to see click on reset password
- User should be able to see a hyperlink to login 

## **What did you do?**
- created the forgot password form component
- created the forgot password page
- integrated the forgot password and signup pages with the backend API endpoints
- created and actions folder and a server action file - **userAuth.ts** that made the API calls
- created a types folder and a types file for  auth - auth.d.ts
- created a config folder

# **Check List**

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR are clear and explain the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit message style matches our requested structure.
- [x] My code additions will not fail code linting checks or unit tests.
- [x] I am only making changes to files I was requested to.

---

# **Screenshots/Videos**

*Add relevant screenshots or videos here if applicable*
screenbud-e232ca8c-a65d-4bdd-a688-00da78cdac84
screenbud-7a047315-2b17-4441-89c4-e6e889b07f24
screenbud-f5b57f7b-3f3e-49e0-bc5e-845512df8dfb

---

### CODE REVIEW

File: .env
Comment (Maiqel1) [inline .env:1]: can't have secrets on the repo, take this out.

**Review by Maiqel1 — CHANGES_REQUESTED** [review]

File: .env
Comment (AdekolaToniloba) [reply .env:1]: I made another commit and added my .env to gitignore, it should not be on the repo

Comment (Maiqel1) [reply .env:1]: but it is.

Comment (AdekolaToniloba) [reply .env:1]: I have removed it

**Review by Maiqel1 — APPROVED** [review]

**Pull request merged**